packages/contracts/test/connect-contracts.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { ethers } from 'hardhat'
+import { Signer, Contract } from 'ethers'
+import { connectL1Contracts, connectL2Contracts } from '../src'
+
+describe('connectL1Contracts', () => {
+  let user: Signer
+  before(async () => {
+    ;[user] = await ethers.getSigners()
+  })
+
+  it('should connect to all mainnet l1 contracts', async () => {
+    const l1Contracts = await connectL1Contracts(user, 'mainnet')
+  })
+})
```

Comment:
Need to add a test for `connectL2Contracts`

Reply:
Not sure if this is enough test coverage. I could also check the addresses are correct, but that would require reaching outside of the test directory (seems like it could cause problems)